tasks.py

```diff
@@ -1104,11 +1104,43 @@ def fetch_dependency(path: str, version: str, fetch_command: str, get_version_co
     run(fetch_command)
     run("chmod +x {}".format(path))
 
-
 def get_command_version(get_version_command: str, version_prefix: str) -> Optional[str]:
     version = run(get_version_command, warn=True, hide='both').stdout
     for line in version.splitlines():
         if line.startswith(version_prefix):
             if ':' in line:
                 return line.split(":")[1].strip()
             return line.split(" ")[1].strip()
+
+def get_frr_k8s_latest():
+    latest = run("curl -s -m 10 --connect-timeout 5 https://api.github.com/repos/metallb/frr-k8s/releases/latest | yq -r .name")
+    if latest == "":
+        raise Exit(message="Failed to fetch latest frr k8s version")
+    return latest
+
+@task
+def checkfrrk8s(ctx):
```

Comment:
This performs the actual bump iiuc. If so, let's call it something like updatefrrk8s